recipes/meta.yaml

```
@@ -10,13 +10,14 @@ package:
 
 source:
   git_url: https://github.com/openghg/openghg.git
-  git_rev: {{ GIT_DESCRIBE_TAG }}
-  git_depth: 1
+  git_rev: {{ GIT_TAG }}
```

```suggestion
  git_rev: {{ GIT_DESCRIBE_TAG }}
```
You were right to remove the `git_depth: 1` line, but I think this should still be `GIT_DESCRIBE_TAG` for the reasons I detailed before. Now that the Git checkout is not limited to one reference, the tags will be present and so `git describe` will work and so `GIT_DESCRIBE_TAG` should contain the correct value.